go.mod

```diff
@@ -4,9 +4,14 @@ require (
 	github.com/adrg/sysfont v0.1.0
 	github.com/boombuler/barcode v1.0.0
 	github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0
+	github.com/gorilla/css v1.0.0 // indirect
```

Comment:
Do we need both of those css libs?

Reply:
github.com/vanng822/css depends of github.com/gorilla/css